### DISCUSSION TITLE
Open AI API's and Keys Help

### DISCUSSION THREAD

**Discussion by saugatacr7** [akash-coded/angular#157] [Q&A]
@akash-coded , Hi Akash, This is Saugata, As discussed on the call, I have to create one Chat GPT type tool, so if you help me to create this APP and help me with Open AI API and Keys.
1. one Chat option will be there like Chat GPT type Search Engine,
2. One UPload Document option will be there, where Document files like pdf, and word can be attached and saved.
3. one Summary option will be there, like if we search for something then it will summarize that thing.
Please if you could help me out.

**Comment by zitraksMD** [comment]
Hello! It's great that you want to create something of your own, like ChatGPT, but there is already so much competition in this area! For example, [AI4Free](https://ai4free.ai/) has already replaced ChatGPT in its entirety, as there is no subscription required, all tools are completely free! Maybe you can take something for your project from this service! Good luck to you!
